examples/platforms/efr32mg1/tradfri/board_config.h

```diff
@@ -0,0 +1,51 @@
+/*
```

Comment:
See my comment reported against the `bspconfig.h` file with regards to replacing the `EFR32MG1_TRADFRI` directory.  Now that I see that there are actually 2 sub-directories, `EFR32MG1_TRADFRI` and `tradfri` I think we should replace the `tradfri` directory with `brd4151a` directory and remove the 'EFR32MG1_TRADFRI` directory.